README.md

```diff
@@ -62,6 +62,11 @@ make install
 ```
 > Note: the application is installed in `/usr/bin` directory and thus the last one command requires root privileges
 
+If you want to make sure that it compiles using Qt5 libraries, select it as the default.
```

Comment:
Do you mean a case when multiple versions are installed? If yes, please clarify.